MaxText/inference/paged_attention.py

```diff
@@ -46,6 +47,7 @@
 D_KV = common_types.D_KV
 
 shard_map = shard_map.shard_map
+use_kernel_v2 = False
```

Comment:
1. Do we want to default to this being False? 
2. If this variable is going to be in use for a while then we should add it to the actual config in base.yml.

Reply:
we could set it to true, my worry is that the new kernel doesn't support SPMD, folks who use > 1 host TPU slice will have issue using this. For 2, the long term goal might be having a flag to choose which kernels to use, but this flag is only temporary.

Reply:
That's fine, let's default to it being false until we feel more comfortable with the state of things.